tests/test_cameras/test_camera_parameter.py

```diff
@@ -52,6 +52,32 @@ def test_load_json():
     assert len(cam_para.get_value('translation')) == 3
 
 
+def test_type():
+    cam_para = CameraParameter(name='src_cam')
+    # wrong distortion value type
+    with pytest.raises(TypeError):
+        cam_para.set_value('k1', '1.0')
+    with pytest.raises(TypeError):
+        cam_para.set_value('k1', np.ones(shape=(3))[0])
+    cam_para.set_value('k1', 1.0)
+    cam_para.reset_distort()
```

Comment:
I think `cam_para.set_value('k1', 1.0)`  would be reset to 0.0 in `cam_para.reset_distort()`, is this intended?

Reply:
Yes.